internal/pkg/docker/docker.go

```diff
@@ -30,9 +30,14 @@ func New() Runner {
 }
 
 // Build will run a `docker build` command with the input uri, tag, and Dockerfile path.
-func (r Runner) Build(uri, imageTag, path string) error {
+func (r Runner) Build(uri, imageTag, path, context string) error {
 	imageName := imageName(uri, imageTag)
-	dfDir := filepath.Dir(path)
+	var dfDir string
+	if context == "" {
+		dfDir = filepath.Dir(path)
+	} else {
+		dfDir = context
+	}
```

Comment:
This seems to be different from what we set up default value for context in [svc_deploy](https://github.com/aws/copilot-cli/pull/1059/files#diff-113644df6967bddf508faa9cdd401b04R375-R377). Do we still want this?

Reply:
This is equivalent to the defaults in svc deploy. I can remove this block or throw an error if it seems like unnecessary implementation sugar, however.